openedx.yaml

```diff
@@ -14,4 +14,16 @@ oeps:
     oep-5:
         state: False
         reason: TODO - Implement for this application if appropriate
+    oep-7:
+        state: False
+        reason: TODO - Implement for this application if appropriate
```

Comment:
Rather than TODOs adding actual tickets to looking into this from your backlog would be useful and more likely to get looked at.

Reply:
@feanil does a ticket exist somewhere in the BOM squad to make edx-proctoring python 3 compatible?

Reply:
There is not a BOM ticket for this yet, looks like it was missed.  I just created https://openedx.atlassian.net/browse/BOM-989 to deal with it since we're on a time crunch otherwise I would ask the owning theme to plan and schedule the work.